cluster/charts/rook-ceph/templates/resources.yaml

```diff
@@ -1333,8 +1333,6 @@ spec:
                   type: object
                   nullable: true
                   x-kubernetes-preserve-unknown-fields: true
-                objectBucketName:
```

Comment:
Instead of a full revert, how about we leave this new setting in the CRD?

Reply:
That's not a bad idea. I can add a commit to the reversion to keep the resource modifications.